docs/data/material/experimental-api/css-variables/CssVariablesCustomization.js

```diff
@@ -22,9 +22,20 @@ const CssVarsCustomButton = styled(Button)({
   '--md-palette-primary-mainChannel': colorChannel('#FF0000'), // necessary for calculating the alpha values
 });
 
+const useEnhancedEffect =
```

Comment:
Why not just importing this from the utils?

Reply:
Because this is the demo and I think it is easier to copy without using `@mui/utils`.